web/src/modules/app/containers/Switcher.js

```diff
@@ -48,15 +47,15 @@ type PropsType = {|
   tunewsLanguagesPayload: Payload<Array<LanguageModel>>,
   tunewsElementPayload: Payload<TunewsModel>,
   offersPayload: Payload<Array<OfferModel>>,
-  sprungbrettJobsPayload: Payload<Array<SprungbrettOfferPage>>,
+  sprungbrettJobsPayload: Payload<Array<typeof SprungbrettOfferPage>>,
```

Comment:
I don't think this type is correct, is it? I think it should be something like `Payload<Array<SprungbrettJobModel>>`

Reply:
fixed